src/main/java/com/spotinst/sdkjava/model/ApiElastigroupStrategy.java

```diff
@@ -0,0 +1,73 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.*;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+
+public class ApiElastigroupStrategy implements IPartialUpdateEntity {
```

Comment:
Rename to ApiElastigroupDeploymentStrategy

Reply:
Done